### ISSUE TITLE
Kanji for 頁　not accepting ぺーじ as correct answer in quiz

### ISSUE THREAD

**Issue by Bigballerbrandball** [davidsansome/tsurukame#778]
Hi, for some reason now the kanji for 頁　is not accepting ぺーじ as correct input/answer. Thanks for the help! 
![IMG_7240](https://github.com/user-attachments/assets/3c0b24a0-b55f-4e6d-9954-fb31bf399689)

**Comment by Bigballerbrandball** [comment]
![IMG_7251](https://github.com/user-attachments/assets/de11c444-9812-4603-bcff-a4e16cdb2b00)

**Comment by Balduran0812** [comment]
> ![IMG_7251](https://private-user-images.githubusercontent.com/189417379/389164720-de11c444-9812-4603-bcff-a4e16cdb2b00.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIzMzY1NzIsIm5iZiI6MTczMjMzNjI3MiwicGF0aCI6Ii8xODk0MTczNzkvMzg5MTY0NzIwLWRlMTFjNDQ0LTk4MTItNDYwMy1iY2ZmLWE0ZTE2Y2RiMmIwMC5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjNUMDQzMTEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Nzc0MmQ5MGQ0MmU2OTJkOWY2ZWZiZTFlZjE3MGU1NzdlN2U3ZjgwZWQ4Y2M0NzQ3MzdiYjM0OTRiYTdhZWNiMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xM4o4SUrFThbbbygz59tFV7rcy8yDA-OXSeEv-qHo5M)

This doesn’t work on my end.

**Comment by shalmaneser** [comment]
Agreed. I haven’t been struggling with this bug all day. I’ve tried every work around I can think of. Copying and pasting exact required katakana for example. Will have to shift back to the Wanikani website until this is fixed.

**Comment by shalmaneser** [comment]
Thank you
